e2e/scripts/st_multiselect.py

```diff
@@ -44,11 +51,16 @@
 i6 = st.multiselect("multiselect 6", options, disabled=True)
 st.text("value 6: %s" % i6)
 
+
+i7 = st.multiselect("choose colors", list(Colors), Colors.yellow)
+
+i7 = st.text("value 7: %s" % i7)
```

Comment:
is this assignment to i7 intentional, or should it be i8?

Reply:
Yeah I put i7 here because the cypress code uses `.last()` to access some of the elements in the test so if I put i8, it caused problems.